### PR TITLE
kill: print ID and state for non-running containers

### PR DESCRIPTION
**Pull request by vrothberg**
Extend kill's error message to include the container's ID and state.
This address cases where error messages caused by other containers
may confuse users.

Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
There's no open issue for that. I just hit that case and wanted to improve the potentially confusing error message a bit.

**Comment by giuseppe** [comment]
LGTM

**Comment by giuseppe** [comment]
and welcome back :-)

File: libpod/container_api.go [hunk 1]
```diff
@@ -203,7 +203,7 @@ func (c *Container) Kill(signal uint) error {
 	}
 
 	if c.state.State != ContainerStateRunning {
-		return errors.Wrapf(ErrCtrStateInvalid, "can only kill running containers")
+		return errors.Wrapf(ErrCtrStateInvalid, "can only kill running containers. %s in state %s", c.ID(), c.state.State.String())
```
Comment (TomSweeneyRedHat) [inline libpod/container_api.go]: suggest s/in state/is in state/

**Comment by TomSweeneyRedHat** [comment]
LGTM, one suggestion for consideration.

File: libpod/container_api.go (hunk 1 above)
Comment (vrothberg) [reply libpod/container_api.go]: Currently, it's consistent with another error. I can change both if you like?

Comment (TomSweeneyRedHat) [reply libpod/container_api.go]: It's a "six of one, half dozen of another" deal.  I prefer "is in", but both are correct.  If you decide to change one, then please do change the other.

Comment (vrothberg) [reply libpod/container_api.go]: Done :heavy_check_mark:

**Pull request merged**